src/qt/qt_mediamenu.cpp

```diff
@@ -156,6 +160,16 @@ void MediaMenu::refresh(QMenu *parentMenu) {
         moMenus[i] = menu;
         moUpdateMenu(i);
     });
+
+    netMenus.clear();
+    MachineStatus::iterateNIC([this, parentMenu](int i) {
+        auto *menu = parentMenu->addMenu("");
+        netDisconnPos = menu->children().count();
+        auto *action = menu->addAction(tr("&Disconnected"), [i] { network_connect(i, !network_is_connected(i)); config_save(); });
```

Comment:
```suggestion
        auto *action = menu->addAction(tr("&Connected"), [i] { network_connect(i, !network_is_connected(i)); config_save(); });
```
Small suggestion: I think it may be less confusing to show the state as "Connected" with a check vs. "Disconnected" with no check. In the default connected state it shows Disconnected without a check, but if you were not aware it can be enabled / disabled it makes it appear as if the current state is Disconnected.